docs/blas.md

```diff
@@ -0,0 +1,162 @@
+<!--
```

Comment:
My recommendation is to name this file "native-backend.md".

"blas.md" doesn't cover the topics.